### DISCUSSION TITLE
Install binary for macOS

### DISCUSSION THREAD

**Discussion by j-zimnowoda** [aquasecurity/kube-bench#766] [Questions and Help] [answered]
Having binary for macOS would ease tool evaluation.

**Comment by mozillazg** [comment]
This project is not working on macOS, so there doesn't have any binary for macOS.

**Comment by lizrice** [comment, marked as answer]
kube-bench needs to run directly on the kubernetes node under test, so that it can access config files and examine running processes. Since kubernetes runs on Linux, this is why we only build kube-bench for Linux
